build.gradle.kts

```diff
@@ -66,6 +70,7 @@ dependencies {
     implementation("com.github.ajalt.clikt:clikt:4.4.0")
     implementation("org.octopusden.octopus.octopus-external-systems-clients:teamcity-client:${properties["teamcity-client.version"]}")
     implementation("org.octopusden.octopus.infrastructure:components-registry-service-client:${properties["components-registry-service-client.version"]}")
+    implementation("org.octopusden.octopus.infrastructure:component-resolver-core:${properties["component-resolver-core.version"]}")

```

Comment:
what do we need it for? crs client should be enough

Reply:
Yes, this dependency is needed for configuring the components registry. We specifically use `ComposedConfigScript` from this dependency within `Aggregator.groovy`, as well as `BuildSystem` and `RepositoryType` for the component configurations. Without it, the components registry service fails to start properly.